allennlp/modules/span_extractors/endpoint_span_extractor.py

```diff
@@ -0,0 +1,72 @@
+
+import torch
+from overrides import overrides
+
+from allennlp.common.params import Params
+from allennlp.modules.span_extractors.span_extractor import SpanExtractor
+from allennlp.modules.token_embedders.embedding import Embedding
+from allennlp.nn.util import batched_index_select, combine_tensors
+from allennlp.data.vocabulary import Vocabulary
+
+@SpanExtractor.register("endpoint")
+class EndpointSpanExtractor(SpanExtractor):
+    """
+    Represents spans as a function of the embeddings of their endpoints. Additionally,
```

Comment:
"as an element-wise function of ...", maybe?  When I first read this I wondered if you allowed parameterized (MLP) functions, but you don't.